config.yaml

```diff
@@ -86,7 +86,10 @@ build:
     destination: ghcr.io/geonet/base-images/chart-centos7:latest
   - source: ./images/rpmbuild-centos7/Dockerfile
     destination: ghcr.io/geonet/base-images/rpmbuild-centos7:latest
+    buildOnMain: true
```

Comment:
is there a suitable way to highlight that this is a restriction to only build on main
vs buildOnAll
onlyBuildOnMain?
buildOnMainOnly?

Reply:
Yeah thought about it for a few minutes. Good for feedback.

How about _requireMainBranch_?

Reply:
Went with _buildOnMainOnly_